host_vars/gcp_postgres_prod1.yml

```diff
@@ -0,0 +1,8 @@
+---
+pg_enabled: false
+node_identifier: 1
+repmgr_master: true
+standby_db_host: gcp-postgres-prod2
+master_db_host: gcp-postgres-prod1
```

Comment:
Maybe `leader_db_host` here instead please? I am sorry, this can be a separate PR, but I just wanted to note this.

Reply:
I will issue and make this a PR. It happens in more than just one place.

Reply:
Created an issue here https://github.com/pulibrary/princeton_ansible/issues/1673